src/orion/core/worker/experiment.py

```diff
@@ -256,6 +256,11 @@ def is_done(self):
 
         .. note:: To be used as a terminating condition in a ``Worker``.
         """
+        self.status = self._db.read('experiments', {'_id': self._id},
```

Comment:
That's a weird side-effect in my opinion. If we want to ensure `status` to be up-to-date then maybe the property method should always call the db.

Reply:
I consider this fixed by #101

Reply:
Indeed! :)